cpp/src/io/comp/nvcomp_adapter.cpp

```diff
@@ -269,6 +269,9 @@ size_t batched_compress_temp_size(compression_type compression,
       compression, num_chunks, max_uncomp_chunk_size, max_total_uncomp_size);
   } catch (...) {
     // Ignore errors in the expanded version; fall back to the old API in case of failure
+    CUDF_LOG_WARN(
+      "Error in CompressGetTempSizeEx call, falling back to CompressGetTempSize; this may increase "
```

Comment:
Suggest not using the word Error in a warning message, to avoid confusion in searches.

```suggestion
      "CompressGetTempSizeEx call failed, falling back to CompressGetTempSize; this may increase "
```